### PR TITLE
R/release 8.0.0 seccom.0

### PR DESCRIPTION
**Pull request by elias-summermatter**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@elias-summermatter I've rebased PR to 8.0.0 release. So we can merge into that one and then back to prod and master.

**Comment by tomaszdurka** [comment]
@elias-summermatter Additionally I can see some new features compared to 8.0.0. 
Are these are all bugfixes from your perspective? If these are not really bugfixes, but missing functionalities we should rather merge these to master and plan yet another release.

**Review by Itrulia — CHANGES_REQUESTED** [review]

File: packages/api/src/graphql/comment/comment.public-queries.ts [hunk 1]
```diff
@@ -50,9 +62,11 @@ export const getPublicCommentsForItemById = async (
   ])
 
   return comments.map(({revisions, ratings, ...comment}) => ({
-    text: revisions[revisions.length - 1].text,
+    title: revisions.length ? revisions[revisions.length - 1].title : null,
```
[the remark this answers is not in the document]
Comment (michael-scheurer) [reply packages/api/src/graphql/comment/comment.public-queries.ts:65]: Current target library in tsconfig doesn't allow it for now. Unfortunately

File: packages/api/src/context.ts [hunk 2]
```diff
@@ -1000,7 +1000,7 @@ export async function contextFromRequest(
         where: {id: payment.id},
         data: {
           state: intent.state,
-          intentID: intent.intentID,
+          intentID: `${intent.intentID}`,
```
[the remark this answers is not in the document]
Comment (michael-scheurer) [reply packages/api/src/context.ts:1003]: This is intended, @elias-summermatter sais.

[the remark this answers is not in the document]
Comment (michael-scheurer) [reply packages/api/src/context.ts:1003]: This is intended, @elias-summermatter sais.

**Comment by michael-scheurer** [comment]
Closing in favor of https://github.com/wepublish/wepublish/pull/745

**Pull request closed without merging**